### ISSUE TITLE
Failing to get transcription for videos that used to work

### ISSUE THREAD

**Issue by nicanor5** [jdepoix/youtube-transcript-api#30]
So as the title says, the module is failing when trying to get the transcript for programs that used to work completely fine before.

Any help would be appreciated.

Edit: I am running Python 3.7.3 on Mac OSX Mojave 10.14.6 but I am also running as a Cloud Function (worked fine until a few days ago).

Here is an example with one of such videos:

```video_id = 'wFdSFVzWeKk'
>>> transcript = YouTubeTranscriptApi.get_transcript(video_id)
Traceback (most recent call last):
  File "/anaconda3/envs/viziotagnlpenv1/lib/python3.7/site-packages/youtube_transcript_api/_api.py", line 93, in get_transcript
    return _TranscriptParser(_TranscriptFetcher(video_id, languages, proxies).fetch()).parse()
  File "/anaconda3/envs/viziotagnlpenv1/lib/python3.7/site-packages/youtube_transcript_api/_api.py", line 160, in parse
    for xml_element in ElementTree.fromstring(self.plain_data)
  File "/anaconda3/envs/viziotagnlpenv1/lib/python3.7/xml/etree/ElementTree.py", line 1315, in XML
    parser.feed(text)
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 2, column 972

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/anaconda3/envs/viziotagnlpenv1/lib/python3.7/site-packages/youtube_transcript_api/_api.py", line 95, in get_transcript
    raise YouTubeTranscriptApi.CouldNotRetrieveTranscript(video_id)
youtube_transcript_api._api.CouldNotRetrieveTranscript: Could not get the transcript for the video https://www.youtube.com/watch?v=wFdSFVzWeKk! This usually happens if one of the following things is the case:
 - subtitles have been disabled by the uploader
 - none of the language codes you provided are valid
 - none of the languages you provided are supported by the video
 - the video is no longer available.

If none of these things is the case, please create an issue at https://github.com/jdepoix/youtube-transcript-api/issues```

**Comment by ArnAV318** [comment]
Facing the same issue with any video which has transcripts.

**Comment by leifoo** [comment]
same issue

**Comment by gunnxx** [comment]
Try to upgrade your package
`pip install --upgrade youtube_transcript_api`

**Comment by jdepoix** [comment]
Fixed with v0.1.9
